### PR TITLE
convert to origami 2.0 component

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by chee** [comment]
@Financial-Times/origami-core

File: origami.json [hunk 1]
```diff
@@ -19,8 +16,8 @@
 		"js": "demos/src/demo.js",
 		"documentClasses": "",
 		"dependencies": [
-			"o-date@^4.0.0",
-			"o-normalise@^2.0.0"
+			"o-date@5.0.0-0",
+			"o-normalise@3.0.0-0"
```
Comment (JakeChampion) [inline origami.json:20]:
```suggestion
			"o-date@^5.0.0",
			"o-normalise@^3.0.0"
```

Comment (JakeChampion) [reply origami.json:20]: This change should make the percy demos be the same as in master

**Review by JakeChampion — COMMENTED** [review]
I approve 👍

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v6.0.0 :tada:

**Pull request merged**